src/styles/blocks/catalog.scss

```diff
@@ -11,15 +14,22 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $hoverTransitionDuration;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale($cardScaleOnHover);
+      .card-title {
+        color: $titleTextColorOnHover;
+      }
```

Comment:
```suggestion
      transform: scale($cardScaleOnHover);
      color: $titleTextColorOnHover;
```

Reply:
Not fixed 
![image](https://github.com/mate-academy/layout_catalog_hovers/assets/99403198/4d9eca4e-96ba-45db-9de3-e08877f1d253)